### DISCUSSION TITLE
Scroll to a specific Row on table initialization

### DISCUSSION THREAD

**Discussion by tektoolsapp** [KevinVandy/material-react-table#836] [General]
In a large table, scroll down many rows then edit row details in another component, then return to table after the edit is completed. The table initializes with the edited row no longer visible.

We have been able to highlight the edited row with a backgroundColor, but can't find a way to auto scroll to a row on initialization.

We can do this is AgGrid with something like this:

const onGridReady = (params) => {
    const { api } = params;

    if (editedRowId) {
      const rowNode = api.getRowNode(editedRowId);
      if (rowNode) {
        rowNode.setSelected(true); // Optionally, select the row
        api.ensureIndexVisible(rowNode.rowIndex);
      }
    }
    localStorage.removeItem('editedCustRowId');
  };

Maybe there is a way?

**Comment by imnasnainaec** [comment]
@tektoolsapp The docs give instructions to [Access Underlying Virtualizer Instances](https://www.material-react-table.com/docs/guides/virtualization#access-underlying-virtualizer-instances), which should allow you to
```
rowVirtualizerInstanceRef?.current.scrollToIndex(rowNode.rowIndex);
```
